doc/source/ray-client.rst

```diff
@@ -0,0 +1,65 @@
+**********
+Ray Client
+**********
+
+.. note::
+
+   This feature is still in beta and subject to changes.
+
+=====
+Basic usage
+=====
+
+While in beta, the server is available as an executable module. To start the server, run
+
+``python -m ray.util.client.server [--host host_ip] [--port port]``
```

Comment:
Can you add a `--ray-address` option here for passing to ray.init()? That way, we don't need a separate "programmatic connection" section below and this can work for both cases.

Reply:
Yes! If we add `--redis-password` too, I can re-use this in https://github.com/ray-project/ray/pull/13217 and don't need an additional file!

Reply:
I changed https://github.com/ray-project/ray/pull/13217 now to be compatible with the way it is done in this PR :)

Reply:
ty! adding the way you implemented to the doc